ci/update_securityhub_suppressions/suppressions.yml

```diff
@@ -112,12 +112,15 @@ Suppressions:
   "S3 bucket server access logging should be enabled":
     - resources:
         - arn:aws:s3:::gdx-vision-dev-tfstate-logs
+        - arn:aws:s3:::gdx-data-share-poc-tfstate-logs
         - arn:aws:s3:::shared-config-logs-gdx-data-share-poc
-        - arn:aws:s3:::shared-vpc-flow-logs-logs-gdx-data-share-poc
         - arn:aws:s3:::grafana-cloudfront-logs
         - arn:aws:s3:::dev-cloudfront-logs-gdx-data-share-poc
-        - arn:aws:s3:::gdx-data-share-poc-tfstate-logs
-        - arn:aws:s3:::demo-vpc-flow-logs-logs-gdx-data-share-poc
-        - arn:aws:s3:::dev-vpc-flow-logs-logs-gdx-data-share-poc
         - arn:aws:s3:::demo-cloudfront-logs-gdx-data-share-poc
+        - arn:aws:s3:::shared-vpc-flow-logs-logs-gdx-data-share-poc
```

Comment:
Silly question, is it worth sorting these alphabetically?

Reply:
I sorted them in sensible groups